src/Aspire.Hosting.Azure/Provisioning/Provisioners/BicepProvisioner.cs

```diff
@@ -152,6 +154,13 @@ await foreach (var kv in keyVaults.GetAllAsync(cancellationToken: cancellationTo
 
             if (keyVault is null)
             {
+                await notificationService.PublishUpdateAsync(resource, state => state with
+                {
+                    State = "Provisioning Keyvault",
+                    StateStyle = "info"
+
+                }).ConfigureAwait(false);
```

Comment:
To make consistent with the code below:

```suggestion
                })
                .ConfigureAwait(false);
```